contracts/WildcardSteward_v2.sol

```diff
@@ -62,15 +61,20 @@ contract WildcardSteward_v2 is Initializable {
         uint256 remainingDeposit
     );
 
-    event AddToken(uint256 indexed tokenId, uint256 patronageNumerator);
+    // Add token generation rate to this event.
+    event AddToken(
+        uint256 indexed tokenId,
+        uint256 patronageNumerator,
+        uint256 tokenGenerationRate
```

Comment:
We must test that this is working fine with thegraph on Goerli, I've never tried having events with the same name but different arguments. 99% sure it works though. But would suck bigtime if we only found out in production.

Reply:
Totally!